migrations/039.do.retrievable-deals.sql

```diff
@@ -0,0 +1,10013 @@
+CREATE TABLE retrievable_deals (
+  cid TEXT NOT NULL PRIMARY KEY,
+  expires_at TIMESTAMPTZ NOT NULL
+);
+
+-- First 10k deals to bootstrap the dataset. Ingest the remaining 22m deals manually.
+
+TRUNCATE TABLE retrievable_deals;
```

Comment:
why truncate it first?

Reply:
Ah, I copy&pasted the SQL generated by fil-deal-ingester. We don't need to truncate the table in this migration script, I'll remove that statement.